app/models/translation_override.rb

```diff
@@ -105,6 +105,21 @@ def self.expire_cache(locale, key)
     true
   end
 
+  # We use English as the source of truth when extracting interpolation keys,
+  #but some languages, like Arabic, have plural forms (zero, two, few, many)
```

Comment:
```suggestion
  # but some languages, like Arabic, have plural forms (zero, two, few, many)
```

Reply:
:grin: I saw that as well and tried really hard to ignore it. :sweat_smile:
I wonder why `stree` doesn't enforce a space at the beginning of comments.

Reply:
Im a lot less annoyed to do this kind of nitpick in reviews now that there are github suggestions, very low hassle for everyone. But yes... it's even better if it's auto :p